guide_drafts/backend_installation.md

```diff
@@ -3,14 +3,14 @@
 Diesel supports SQLite, PostgreSQL, and MySQL as database backends.
 By default, `diesel_cli`
 requires the client library of all three backends to be installed.
-If one is missing, then `cargo install diesel_cli` you will throw an error like:
+If one is missing, then `cargo install diesel_cli` you will receive an error like:
```

Comment:
"…then `foo` you will receive" doesn't read like English to me. Wasn't the original meant like

> then `cargo install diesel_cli` will throw (you) an error like